launch/tier4_planning_launch/launch/scenario_planning/lane_driving/behavior_planning/behavior_planning.launch.py

```diff
@@ -384,39 +384,57 @@ def launch_setup(context, *args, **kwargs):
         output="screen",
     )
 
-    # load compare map for dynamic obstacle stop module
+    # This condition determine whether the points filter below are launched
+    launch_run_out_with_points_method = PythonExpression(
+        [
+            LaunchConfiguration(
+                "launch_run_out", default=behavior_velocity_planner_param["launch_run_out"]
+            ),
+            " and ",
+            "'",
+            run_out_param["run_out"]["detection_method"],
+            "' == 'Points'",
+        ]
+    )
+
```

Comment:
@yukkysaito 
I think no_detection_filter should be always launched 
because no_detection_filter is useful not only for run_out module but also other modules such as pull_over.
What do you think?

Reply:
Thank you! You are right. But there is a complex problem for our products.
We do not want to use this as an output of Perception. However, there are cases where we want to use it for the convenience of run-out module.

@TomohitoAndo @mkuri 
Can you disscus this?

Reply:
discussion result

https://github.com/autowarefoundation/autoware.universe/pull/1530#issuecomment-1220541843